python/dask_cudf/dask_cudf/io/tests/test_parquet.py

```diff
@@ -309,6 +310,27 @@ def test_roundtrip_from_dask_partitioned(tmpdir, parts, daskcudf, metadata):
         dd.assert_eq(df_read, gdf_read)
 
 
+@pytest.mark.parametrize(
+    "partition_on,row_group_cols",
+    [("year", "month"), ("year", "data"), ("data", "year")],
+)
+def test_row_group_partitioned(tmpdir, partition_on, row_group_cols):
+    tmpdir = str(tmpdir)
+
+    df = pd.DataFrame()
+    df["year"] = [2018, 2019, 2019, 2019, 2020, 2021]
+    df["month"] = [1, 2, 3, 3, 3, 2]
+    df["day"] = [1, 1, 1, 2, 2, 1]
+    df["data"] = [0, 0, 0, 0, 0, 0]
+
+    ddf = dask_cudf.from_cudf(cudf.from_pandas(df), npartitions=2)
+    ddf.to_parquet(
+        tmpdir, partition_on=partition_on, row_group_cols=row_group_cols
+    )
+    ddf_read = dask_cudf.read_parquet(tmpdir)
+    assert_eq(len(ddf), len(ddf_read))
```

Comment:
`assert_eq` is intended for more complicated assertions about dataframes being equivalent. Comparing lengths should be done with a plain `assert`. However, it is probaby a good idea to make sure the data frame written/read is equivalent to the source dataframe in memory:
```suggestion
    assert len(ddf) == len(ddf_read)
    assert_eq(ddf, ddf_read)
```